dropshot/src/handler.rs

```diff
@@ -139,6 +150,21 @@ impl<Context: ServerContext> RequestContext<Context> {
              */
             .unwrap_or(server_config.page_default_nitems))
     }
+
+    /**
+     * Add a header that will be included with the HTTP response.
+     */
+    pub fn add_response_header(
+        self: Arc<Self>,
+        header: http::header::HeaderName,
```

Comment:
nit, and *please* feel free to ignore: I see that `HeaderMap::insert()` takes `IntoHeaderName`, and I wonder if that's not more general in a useful way.  We could even expose a `&mut HeaderMap` directly to people and avoid dealing with a bunch of these issues ourselves.

Reply:
That would make it much easier to use. Good idea.

Reply:
I agree with Dave here. Let `HeaderMap` handle issues of conversion, duplicate values, etc.